packages/inference/src/types.ts

```diff
@@ -127,6 +124,6 @@ export interface UrlParams {
 export interface BodyParams {
 	args: Record<string, unknown>;
 	chatCompletion?: boolean;
-	model: string;
+	model?: string;
```

Comment:
I would prefer if we keep `model: string` here since not passing a model should be considered the exception. Since `model` is undefined only when endpoint is passed, in which case "hf-inference"` is used, can we define a specific interface used only by `"hf-inference"`? Doing this, we won't have to touch any other providers' logic